electron/src/chatBro/queries/TopFriends/filters.ts

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+import { Columns } from './columns';
+
+function wordFilter(filters: TopFriendsTypes.Filters): string | undefined {
+  if (_.isEmpty(filters.word)) {
+    return undefined;
+  }
+  // NOTE: using LIKE because TopFriends query is not split word by word
+  return `LOWER(${Columns.TEXT}) LIKE "%${filters.word?.toLowerCase()}%"`;
+}
+
+function contactFilter(filters: TopFriendsTypes.Filters): string | undefined {
```

Comment:
minor nit: Think the filters should just have what's needed for each specific one vs passing in all filters.

Reply:
Typescript is complaining and I can't seem to figure out how to fix it, let's table this and fix it later.